packages/agents/src/agents/Evo/functions/DelegateScriptedAgent.ts

```diff
@@ -37,8 +38,14 @@ export class DelegateAgentFunction<
         task: {
           type: "string",
           description: "The task to be delegated"
+        },
+        context: {
+          type: "string",
+          description: "Necessary information required to completed the task."
         }
-      }
+      },
+      required: ["task"],
```

Comment:
Why not put context required?

Reply:
Because the task can be simple and provide enough information. I haven't seen a problem with it being optional so I'm going to leave it for now. Can change in the future if it isn't using it effectively.